ext/console/02_console.js

```diff
@@ -1650,7 +1650,7 @@
   }
 
   function inspectArgs(args, inspectOptions = {}) {
-    const noColor = globalThis.Deno?.noColor ?? true;
```

Comment:
I'm don't understand why was this defaulting to `true` while all other coercions defaulted to false.